docs/api-docs/apps/apps-quick-start.mdx

```diff
@@ -91,19 +94,25 @@ FIRE_API_KEY={firebase key}
 FIRE_DOMAIN={firebase domain}
 FIRE_PROJECT_ID={firebase project id}
 
-# If using mysql, enter your config here
+# If using mysql, enter your config here and change DB_TYPE to mysql
 MYSQL_HOST={mysql host}
 MYSQL_DATABASE={mysql database name}
 MYSQL_USERNAME={mysql username}
 MYSQL_PASSWORD={mysql password}
 MYSQL_PORT={mysql port *optional*}
+
+# Most users do not need to change this
+ENVIRONMENT=bigcommerce.com
+LOGIN_URL=login.${ENVIRONMENT}
+API_URL=api.${ENVIRONMENT}
```

Comment:
these variables are for internal testing purposes only - these should not be listed for public consumption

Reply:
`API_URL` is required to use app extensions, so I've retained that value.